bin/p2-exec/main.go

```diff
@@ -37,6 +38,8 @@ func main() {
 	kingpin.Version(version.VERSION)
 	kingpin.Parse()
 
+	syscall.Umask(*umask)
```

Comment:
New code should use the ["golang.org/x/sys/unix"](https://godoc.org/golang.org/x/sys/unix) package instead of "syscall".